Core/View/AdminPlugins.html.twig

```diff
@@ -25,6 +25,41 @@
 
 {% extends "Master/MenuTemplate.html.twig" %}
 
+{% block javascripts %}
+    {{ parent() }}
+    <script type="text/javascript">
+        //Function to show confirmation popup to remove a plugin
+        function deletePlugin(pluginName) {
+            // This is a workaround because we can't translate in real time.
+            var msg = "{{ i18n.trans('are-you-sure-plugin', {'%pluginName%': 'placeholder'}) }}";
+            msg = msg.replace('placeholder', pluginName);
+
+            bootbox.confirm({
+                title: "{{ i18n.trans('confirm-delete')|raw }}",
+                message: msg,
+                closeButton: false,
+                buttons: {
+                    cancel: {
+                        label: "<i class='fa fa-times'></i> {{ i18n.trans('cancel') }}"
+                    },
+                    confirm: {
+                        label: "<i class='fa fa-check'></i> {{ i18n.trans('delete') }}",
+                        className: 'btn-danger'
+                    }
+                },
+                callback: function (result) {
+                    if (result) {
+                       execActionForm(pluginName);
+                    }
+                }
+            });
+        }
+        function execActionForm(pluginName) {
```

Comment:
Unnecesary function. Move:

window.location = "{{fsc.url()}}?action=remove"+"\u0026"+"plugin=" + pluginName;

To deletePlugin()